hepdata/modules/records/templates/hepdata_records/components/record-javascript.html

```diff
@@ -16,6 +16,7 @@
 
         HEPDATA.upload_max_size = {{ config['UPLOAD_MAX_SIZE']|default(52000000) }};
         HEPDATA.upload_timeout = {{ config['NGINX_TIMEOUT']|default(298) }};
+        HEPDATA.show_review = {{ 'true' if ctx.show_review_widget or ctx.show_upload_widget else 'false'}};
```

Comment:
Should the `true` condition also check for `ctx.status == 'todo'` to avoid making unnecessary calls for Sandbox or finished records?

Reply:
Yes - now done.